src/Symfony/Component/Form/Extension/DataCollector/EventListener/DataCollectorListener.php

```diff
@@ -34,8 +34,8 @@ public function __construct(FormDataCollectorInterface $dataCollector)
     public static function getSubscribedEvents(): array
     {
         return [
-            // High priority in order to be called as soon as possible
-            FormEvents::POST_SET_DATA => ['postSetData', 255],
+            // Low priority in order to be called as late as possible
```

Comment:
Do you know why the priority was really high before? Could it be that we will forbid some use case with these changes?

Reply:
Not really, this is there since the origin (#9082).
There are not core usage of post set data event (except for a recent deprecation concerning timezone :)).
But the changes in this PR tells me that if any listener out there modifies the form structure, we would collect a tree out of sync and may display irrelevant information in the profiler.